packages/core/src/storage/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './constants';
 export * from './mock';
 export * from './domains';
 export * from './utils';
+export * from './sql-builder';
```

Comment:
can you create a new export in core instead.
so`@mastra/core/storage/sql-builder`